rust/main/agents/relayer/src/msg/metadata/aggregation.rs

```diff
@@ -134,13 +133,11 @@ impl MetadataBuilder for AggregationIsmMetadataBuilder {
             .base_builder()
             .build_aggregation_ism(ism_address)
             .await
-            .context(CTX)
-            .map_err(|_| MetadataBuildError::FailedToBuild)?;
+            .map_err(|_| MetadataBuildError::FailedToBuild(CTX.into()))?;
```

Comment:
what I had in mind is something like this, so we can see the actual error messages. Everywhere else as well
```suggestion
            .map_err(|err| MetadataBuildError::FailedToBuild(err.to_string()))?;
```

Reply:
ah I see. Let me change it xd